apps-rendering/cdk/lib/mobile-apps-rendering.ts

```diff
@@ -84,7 +84,7 @@ export class MobileAppsRendering extends GuStack {
 			},
 			instanceType: InstanceType.of(
 				InstanceClass.T4G,
-				InstanceSize.MICRO,
+				InstanceSize.SMALL,
```

Comment:
this changes the instance size for both the live AR and the one which reads from preview - would we want to keep the instance size for the preview deployment as micro?

I'd imagine because the preview deployment receives less traffic it is less likely to be experiencing the same problem, though I can see some advantages to keeping them the same.

Reply:
Good question. If it's simple to provision preview separately in CDK maybe we should keep it on the smaller size?

What do you think @jacobwinch ?

Reply:
#5966